root/static/scripts/edit/URLCleanup.js

```diff
@@ -2598,13 +2502,10 @@ const CLEANUPS: CleanupEntries = {
        * sk (subpages in a page, since we want the main link)
        * and a couple others
        */
-      url = url.replace(new RegExp(
-        '([&?])(__tn__|_fb_noscript|_rdr|acontext|em|entry_point|filter|' +
-        'focus_composer|fref|hc_location|pnref|qsefr|ref|' +
-        'ref_dashboard_filter|ref_page_id|ref_type|refsrc|rf|' +
-        'sid_reminder|sk|tab|viewas)=([^?&]*)',
-        'g',
-      ), '$1');
+      url = url.replace(
+        /([&?])(__tn__|_fb_noscript|_rdr|acontext|em|entry_point|filter|focus_composer|fref|hc_location|pnref|qsefr|ref|ref_dashboard_filter|ref_page_id|ref_type|refsrc|rf|sid_reminder|sk|tab|viewas)=([^?&]*)/g,
```

Comment:
Seems like an improvement in the majority of cases, but maybe exceptions can be added for cases like this, where having it across multiple lines is easier to read, and we aren't benefiting from reduced escaping.

Reply:
Turns out this one isn't even caught by the rule so we don't need an exception.